### PR TITLE
Updates in README as recommended by cathsantos@

### PR DESCRIPTION
**Pull request by chitara-01**
Updating README with final review changes provided by cathsantos@.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -170,9 +177,11 @@ gradle run -DmainClass=com.google.swarm.tokenization.DLPTextToBigQueryStreamingV
 --serviceAccount=${SERVICE_ACCOUNT_EMAIL}"
 ```
 
-Above command will trigger a batch inspection Dataflow pipeline that will process only the CSV files that were present in the demo-data GCS bucket (specify in _filePattern_ parameter) before the pipeline was triggered.
+Any files that are added to the bucket after the pipeline was triggered are not inspected.
+
+##### Inspect only new files
 
-Run following command to trigger the inspection pipeline to process only the new files added to the GCS bucket(files added after the pipeline is triggered):
+To trigger a streaming inspection Dataflow pipeline that processes new CSV files that are added to the `demo-data` bucket (specified in `filePattern` parameter), run the following command:
```
Comment (Goutam1511) [inline README.md:184]: Can we mention the full name of the bucket which would be <PROJECT_ID>-demo-data?

File: README.md [hunk 2]
```diff
@@ -193,24 +202,33 @@ gradle run -DmainClass=com.google.swarm.tokenization.DLPTextToBigQueryStreamingV
...
+You can run the de-identification pipeline to do one or both of the following:
 
-The De-Identification pipeline can either be run to deidentify only the existing files in a GCS bucket or deidentify only the new files being added to the bucket after the pipeline is triggered or do both.
+* De-identify only existing files in the Cloud Storage bucket.
+* De-identify only new files added to the Cloud Storage bucket after the pipeline is triggered.
 
-Run following command to trigger the de-identification pipeline for both existing files in the GCS bucket and the new files that will be added to the bucket after the pipeline is triggered:
+##### De-identify new and existing files
+
+To trigger a streaming de-identification Dataflow pipeline that processes all the CSV files in the `demo-data` bucket (specified in the `filePattern` parameter), run the following command:
```
Comment (Goutam1511) [inline README.md:231]: full name of the bucket again

File: README.md [hunk 3]
```diff
@@ -230,9 +248,9 @@ gradle run -DmainClass=com.google.swarm.tokenization.DLPTextToBigQueryStreamingV
 --gcsNotificationTopic=projects/${PROJECT_ID}/topics/${GCS_NOTIFICATION_TOPIC}"
 ```
 
-Above command will trigger a streaming de-identification Dataflow pipeline that will process all the CSV files in the demo-data GCS bucket (specify in _filePattern_ parameter) - both the existing files and the new files added to the bucket after triggering the pipeline.
+##### De-identify only existing files
 
-Run following command to trigger the de-identification pipeline to process only the existing files in the GCS bucket(files present before the pipeline is triggered):
+To trigger a batch de-identification Dataflow pipeline that processes only the CSV files that are currently present in the `demo-data` bucket (specified in the `filePattern` parameter), run the following command:
```
Comment (Goutam1511) [inline README.md:253]: same bucket name please

File: README.md [hunk 4]
```diff
@@ -250,9 +268,11 @@ gradle run -DmainClass=com.google.swarm.tokenization.DLPTextToBigQueryStreamingV
 --serviceAccount=${SERVICE_ACCOUNT_EMAIL}"
 ```
 
-Above command will trigger a batch de-identification Dataflow pipeline that will process only the CSV files that were present in the demo-data GCS bucket (specify in _filePattern_ parameter) before the pipeline was triggered.
+Any files that are added to the bucket after the pipeline was triggered are not de-identified.
+
+##### De-identify only new files
 
-Run following command to trigger the de-identification pipeline to process only the new files added to the GCS bucket(files added after the pipeline is triggered):
+To trigger a streaming de-identification Dataflow pipeline that processes new CSV files that are added to the `demo-data` bucket (specified in `filePattern` parameter), run the following command:
```
Comment (Goutam1511) [inline README.md:275]: same as above - bucket name please

**Review by Goutam1511 — APPROVED** [review]
Approving but minor changes suggested. Can you kindly fix them as well?

**Review by annadata — APPROVED** [review]

**Review by Goutam1511 — APPROVED** [review]

**Pull request merged**